system/loggerd/loggerd.h

```diff
@@ -39,7 +42,7 @@ class EncoderInfo {
   int frame_width = -1;
   int frame_height = -1;
   int fps = MAIN_FPS;
-  int bitrate = MAIN_BITRATE;
```

Comment:
it should probably always be a callback

Reply:
but wouldn't that also need a separate callback for gop size too? I don't like the idea of defining a pair attribute which is only relevant for h265s